index.bs

```diff
@@ -492,7 +492,7 @@ The <dfn attribute for="XRSession">environmentBlendMode</dfn> attribute returns
 
 NOTE: Most Virtual Reality devices exhibit {{XREnvironmentBlendMode/opaque}} blending behavior. Augmented Reality devices that use transparent optical elements frequently exhibit {{XREnvironmentBlendMode/additive}} blending behavior, and Augmented Reality devices that use passthrough cameras frequently exhibit {{XREnvironmentBlendMode/alpha-blend}} blending behavior.
 
-Each {{XRSession}} has a <dfn for="XRSession">viewer space</dfn>, which is an {{XRReferenceSpace}} with a [=native origin=] which tracks the position and orientation of the [=viewer=]. The [=XRSession/viewer space=] has a <dfn for="XRSession/viewer space">list of views</dfn>, which is a [=/list=] of [=view=]s corresponding to the views provided by the [=/XR device=].
+Each {{XRSession}} has a <dfn for="XRSession">viewer space</dfn>, which is an {{XRReferenceSpace}} of type {{XRReferenceSpaceType/viewer}} with an [=identity transform=] [=XRSpace/origin offset=]. The [=XRSession/viewer space=] has a <dfn for="XRSession/viewer space">list of views</dfn>, which is a [=/list=] of [=view=]s corresponding to the views provided by the [=/XR device=].
```

Comment:
Should we be calling it a "viewer reference space"? It'd be so long that way....

Reply:
Meh, it's not all that bad. Made the change for clarity's sake.